dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/ConnectorArguments.java

```diff
@@ -717,6 +717,10 @@ public boolean isDryRun() {
     return getOptions().has(optionDryrun);
   }
 
+  public boolean isLogConnector() {
```

Comment:
Do we really need such method? I think we can use this one `isDefinitionSpecified`

Reply:
`isDefinitionSpecified` checks if property was specified. Here I want to know, if we are running query log assessment, by checking connector. Even if `query-log-start` is not specified, but it is related to query log connector, we want to output message.

Reply:
I see. Sorry, I misunderstand the `getConnectorName()` call.

Is the containing in name of the connector `logs` robust enough? Maybe it will better to introduce a flag/property to connectors or has an explicit list of connectors?

Reply:
I checked that all logs connector contains `log` in the name, so they are following that pattern.
Having explicit list connectors, means that, if we add new log connector, that need to be updated as well. Can be option as well. What other developers think? Should I add general set with all log connectors name or do you think there could be better approach to check if connector is log related?

Reply:
If there is an existing naming convention that everyone should follow, that's fine. I just haven't seen any checks in the code that could prevent future bugs.
However, I don't have enough knowledge about the project yet. Let's wait for other opinions.